### ISSUE TITLE
Update active grouping and action layout

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#1194]
**Context**
- As part of https://github.com/pixiebrix/pixiebrix-extension/issues/1137 we're moving some extension options to the "Active Bricks" page (under a "more"-style dropdown)
- In https://github.com/pixiebrix/pixiebrix-extension/issues/1150 we added support for cloud-synced extensions. So the page will show some extensions that aren

**Wireframes**
- See "My Bricks" page https://balsamiq.cloud/s7ffzg5/pu105es

**Implementation Sketch**
- Split into the 3 groups: 
  - personal: do not have a _recipe set
  - marketplace: have a _recipe set
  - deployments: have a _deployment set
- Put all actions behind dropdowns similar to #1137 

**Notes**
- Deployments are not expandable
- Team/Public marketplace entries start collapsed

**Out of Scope**
We'll have follow up tickets for the following:
- The Log popup: https://github.com/pixiebrix/pixiebrix-extension/issues/1196
- The Share Popup
- Splitting out Team vs. Public marketplace bricks. (Currently blocked by: https://github.com/pixiebrix/pixiebrix-app/issues/514)
